chamge/amqp-arbiter-backend.c

```diff
@@ -32,6 +32,43 @@ struct _ChamgeAmqpArbiterBackend
 G_DEFINE_TYPE (ChamgeAmqpArbiterBackend, chamge_amqp_arbiter_backend, CHAMGE_TYPE_ARBITER_BACKEND)
 /* *INDENT-ON* */
 
+inline static const gchar *
+_amqp_get_rpc_reply_string (amqp_rpc_reply_t r)
+{
+  if (r.reply_type == AMQP_RESPONSE_NORMAL) {
+    return "response normal";
+  } else if (r.reply_type == AMQP_RESPONSE_NONE) {
+    return "reply type is missing";
+  } else if (r.reply_type == AMQP_RESPONSE_LIBRARY_EXCEPTION) {
+    return amqp_error_string2 (r.library_error);
+  } else if (r.reply_type == AMQP_RESPONSE_SERVER_EXCEPTION) {
+    int ret = 0;
+    static char str[512] = { 0 };
```

Comment:
For this case, IMO, it is better to return string by `g_strdup()` or its friend to prevent programmer error.
But not for this time.